### ISSUE TITLE
Tests failing in V13 because of Test Replay with canvas interaction

### ISSUE THREAD

**Issue by whitespacecode** [cypress-io/cypress#29042]
### Current behavior

As stated in ticket https://github.com/cypress-io/cypress/issues/28289 you asked me to create a new ticket.

We have a canvas with pixijs elements. We can click these elements to do some actions.
Since V13 with Test Replay the canvas is gone? And the tests also don't work

Failing on V13
https://cloud.cypress.io/projects/yzkmsw/runs/421

Reverting to V12 fixes the issue

### Desired behavior

View the canvas ?

### Test code to reproduce

https://cloud.cypress.io/projects/yzkmsw/runs/421

### Cypress Version

13.6.6

### Node version

18.18.0

### Operating System

Ubuntu

### Debug Logs

```shell
[4034:0301/091647.479590:ERROR:gl_utils.cc(402)] [.WebGL-0x2640029aa00]GL Driver Message (OpenGL, Performance, GL_CLOSE_PATH_NV, High): GPU stall due to ReadPixels
    ✓ Configuration page should be accessible (11074ms)
[4034:0301/091649.530710:ERROR:gl_utils.cc(402)] [.WebGL-0x264044bd500]GL Driver Message (OpenGL, Performance, GL_CLOSE_PATH_NV, High): GPU stall due to ReadPixels
[4034:0301/091650.659607:ERROR:gl_utils.cc(402)] [.WebGL-0x264044bd500]GL Driver Message (OpenGL, Performance, GL_CLOSE_PATH_NV, High): GPU stall due to ReadPixels
[4034:0301/091650.829657:ERROR:gl_utils.cc(402)] [.WebGL-0x264044bd500]GL Driver Message (OpenGL, Performance, GL_CLOSE_PATH_NV, High): GPU stall due to ReadPixels (this message will no longer repeat)
    ✓ marker click (3326ms)
    1) Click floor in canvas
    ✓ Navigation (1499ms)
    ✓ Save configuration (7924ms)


  4 passing (33s)
  1 failing

  1) Configuration page
       Click floor in canvas:
     AssertionError: Timed out retrying after 4000ms: Expected to find element: `.outline`, but never found it.
```

**Comment by jennifer-shehane** [comment]
@whitespacecode Could you try a few things to narrow down the root cause? There's a few differences between the runs that pass on 12 and the runs that don't on 13. 

- Can you run the test on v13.6.6 with Test Replay disabled? What is the effect of this? https://docs.cypress.io/guides/cloud/test-replay#Opt-out-of-Test-Replay
- After turning off Test Replay, if the issue is still occuring, could you try to record a video on the run to see the changed behavior. (When we have canvas in Test Replay this will be easier to debug). 
- Can you run the test on v13.6.6 with the Chrome browser (`--browser chrome`) and various versions? Links [here](https://vikyd.github.io/download-chromium-history-version/#/) to download. In the v12 version, you're running on Electron 106, which uses Chromium 106. In 13.6.6 you're running on Electron 118, which means it's running on Chromium 118. There's some differences between these browsers, so want to exclude Chrome introducing a change that caused this. 

Thanks for opening up the issue!

**Comment by whitespacecode** [comment]
@jennifer-shehane 

We upgraded to V13.6.6

- They only in Github actions, testing locally we have no issues. 
- The tests fail when using Electron with Test Replay enabled. When disabling Test Replay the tests also fail.
- When running the tests with the latest version of Chrome and Test Replay enabled, they pass successfully.
- For our GitHub actions, we utilize `browser: chrome` to incorporate Chrome. Manually installing Chromium versions is too cumbersome and impractical for our workflow.

So tldr; Replay works with the latest chrome browser in V13 everything else fails.

Setup:
```
- name: Cypress run
  uses: cypress-io/github-action@v6
  with:
    build: npm run build
    start: npm start
    record: true
    browser: chrome
```

**Comment by jennifer-shehane** [comment]
@whitespacecode Were you able to find a resolution to this? Have you tried a later version of Cypress?

**Comment by jennifer-shehane** [comment]
Closing since there is no response.
